lib/generators/shopify_app/rotate_shopify_token_job/templates/rotate_shopify_token.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+namespace :shopify do
+  desc "Rotate shopify tokens for all active shops"
+  task :rotate_shopify_tokens, [:refresh_token] => :environment do |_t, args|
+    all_active_shops.each do |shop|
+      Shopify::RotateShopifyTokenJob.new.perform(
```

Comment:
Just to confirm: Is this being run asynchronously, or in the loop?

Reply:
Good catch – this should be async